docs: add note about disabling ESM related MOTD messages (#2939)
canonical/ubuntu-pro-client: docs/explanations/motd_messages.md

2. That hook will update the information in `/var/lib/update-notifier/updates-available` matching the new package information that was just fetched by using `/usr/lib/update-notifier/apt-check --human-readable`.
3. At MOTD generation time, the script located at `/etc/update-motd.d/90-updates-available` checks if `/var/lib/update-notifier/updates-available` exists and if it does, inserts the message into the full MOTD.

> I you want to remove the messages about Ubuntu Pro and ESM from the MOTD

Reviewer: ```suggestion
If you want to remove the messages about Ubuntu Pro and ESM from the MOTD
```

Author: lol me blind sorry

> output while keeping the messages about the regular and security updates
> around, edit `/etc/update-motd.d/90-updates-available` adding the `--no-esm-messages` flag to all calls of the update script. For example, change:

Reviewer: ```suggestion
output, but still want to keep keep the messages about regular and security updates, edit the `/etc/update-motd.d/90-updates-available` file. Add the `--no-esm-messages` flag to all calls of the update script. For example, change following line from:
```

```sh
APT::Update::Post-Invoke-Success {"/usr/lib/update-notifier/update-motd-updates-available 2>/dev/null || true";};
```

to:

```sh
APT::Update::Post-Invoke-Success {"/usr/lib/update-notifier/update-motd-updates-available --no-esm-messages 2>/dev/null || true";};
```

> If you want to disable any message of update-notifier (not just related to Ubuntu Pro and ESM) about potentially available updates, just remove `/etc/update-motd.d/90-updates-available`.

Reviewer: ```suggestion
If you want to disable all messages from update-notifier (not just related to Ubuntu Pro and ESM) about potentially available updates, just remove the file `/etc/update-motd.d/90-updates-available`.
```
